contrib/ansible/roles/kubernetes-addons/templates/kube-addons.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
```

Comment:
I'm guessing this is cp'd from somewhere else in the tree?  does git support symlinks?  I don't want this getting out of date...